plugins/inputs/netflow/README.md

```diff
@@ -0,0 +1,101 @@
+# Netflow Input Plugin
+
+The `netflow` plugin acts as a collector for Netflow v5, Netflow v9 and IPFIX
+flow information. The Layer 4 protocol numbers are gathered from the
+[official IANA assignments][IANA assignments].
+The internal field mappings for Netflow v5 fields are defined according to
+[Cisco's Netflow v5 documentation][CISCO NF5], Netflow v9 fields are defined
+according to [Cisco's Netflow v9 documentation][CISCO NF9] and the
+[ASA extensions][].
+Definitions for IPFIX are according to [IANA assignement document][IPFIX doc].
+
+[IANA assignments]: https://www.iana.org/assignments/protocol-numbers/protocol-numbers.xhtml
+[CISCO NF5]:        https://www.cisco.com/c/en/us/td/docs/net_mgmt/netflow_collection_engine/3-6/user/guide/format.html#wp1006186
+[CISCO NF9]:        https://www.cisco.com/en/US/technologies/tk648/tk362/technologies_white_paper09186a00800a3db9.html
+[ASA extensions]:   https://www.cisco.com/c/en/us/td/docs/security/asa/special/netflow/asa_netflow.html
+[IPFIX doc]:        https://www.iana.org/assignments/ipfix/ipfix.xhtml#ipfix-nat-type
+
+## Configuration
+
+```toml @sample.conf
+# Netflow v5, Netflow v9 and IPFIX collector
+[[inputs.netflow]]
+  ## Address to listen for netflow/ipfix packets.
+  ##   example: service_address = "udp://:2055"
+  ##            service_address = "udp4://:2055"
+  ##            service_address = "udp6://:2055"
+  service_address = "udp://:2055"
+
+  ## Set the size of the operating system's receive buffer.
+  ##   example: read_buffer_size = "64KiB"
+  # read_buffer_size = ""
```

Comment:
Is this the OS buffer or our buffer? Does this mean we default to no buffer? If this is set, does the user need elevated permissions to change it? Any considerations a user should know about this?

Reply:
It defaults to the OS buffer size if not given, but you can overwrite it `conn.SetReadBuffer(int(n.ReadBufferSize))` with this setting. Basically copied this from `socket_listener`...

Reply:
Can you put where the default comes from in the comment?

Reply:
Done in the update.